core/modules/system/system.module

```diff
@@ -3808,21 +3808,6 @@ function system_image_toolkits() {
   );
 }
 
-/**
- * Implements hook_image_supported_extensions_alter().
- *
- * Add webp to supported file extensions if GD is the active image toolkit and
- * the GD version supports WebP.
- */
-function system_image_supported_extensions_alter(&$supported_extensions) {
```

Comment:
@quicksketch can we remove an API without deprecating it first?  This seems like it will be problematic.